lambda/locals.tf

```diff
@@ -3,6 +3,7 @@ locals {
     var.layers,
     var.enable_lambda_insights ? [
       "arn:aws:lambda:${data.aws_region.current.name}:580247275435:layer:LambdaInsightsExtension:${var.lambda_insights_version}",
-    ] : []
+    ] : [],
+    var.enable_lambda_layers_rds_ca ? [ module.lambda_layer_rds_ca.arn ] : []
```

Comment:
The _module.lambda_layer_rds_ca_ module will have an array, so access via the array object is missing here: should be module.lambda_layer_rds_ca[0].arn